packages/eslint-config-4catalyzer/rules.js

```diff
@@ -78,4 +84,13 @@ module.exports = {
     },
   ],
   'import/prefer-default-export': 'off',
+  'import/order': [
+    'error',
+    {
+      'newlines-between': 'never',
```

Comment:
@taion and @jquense let's forget for a moment this is broken due to issue above... do you think should be `always` or `always-inside-groups` ([doc](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#newlines-between-ignorealwaysalways-and-inside-groupsnever))

Reply:
i think we want newlines between groups, but not within groups?

Reply:
so `always`? sure. yeah. if there is a reason to override, is ok to `eslint-ignore`